sdk/nodejs/iot/zMixins.ts

```diff
@@ -14,7 +14,7 @@ declare module "./ioTHub" {
     }
 }
 
-IoTHub.prototype.onEvent = function(this: IoTHub, name, args, opts) {
+IoTHub.prototype.onEvent = function (this: IoTHub, name, args, opts) {
```

Comment:
Nit: No need in space after `function`

Reply:
Fixed in 4befc8cc16a7a663e608029b69f0fd4a8d66b1e2